utils/demo/helpers/addManipulationBindings.ts

```diff
@@ -22,7 +23,24 @@ const { MouseBindings, KeyboardBindings } = csToolsEnums;
  * * Stack Scroll on Mouse Wheel, Primary+Alt
  * * Length Tool on fourth button
  */
-export default function addManipulationBindings(toolGroup, register = true) {
+export default function addManipulationBindings(
+  toolGroup,
+  register = true,
+  options?
+) {
+  const zoomBindings: Types.IToolBinding[] = [
+    {
+      mouseButton: MouseBindings.Secondary,
+    },
+  ];
+
+  if (options?.disableShiftClickZoom !== true) {
+    zoomBindings.push({
+      mouseButton: MouseBindings.Primary, // Shift Left Click
+      modifierKey: KeyboardBindings.Shift,
+    });
+  }
```

Comment:
I'm not a fan of this being the default setting. Let's completely remove it. Instead of having an option to disable it, we should have an option to enable it for those who are interested.